shared/components/CameraView.js

```diff
@@ -108,6 +108,20 @@ class CameraView extends Component {
     };
   }
 
+  componentDidMount() {
+    navigator.geolocation.getCurrentPosition(
```

Comment:
well, a user can disable GPS after running our app. So, checking this again in cameraView will stop the user from disabling GPS in-between